services/bigo_live.yml

```diff
@@ -0,0 +1,8 @@
+id: bigo_live
+name: Bigo Live
+rules:
+  - '||bigolive.tv^'
+  - '||bigo.tv^'
+  - '||bigovideo.tv^'
+  - '||bigo.sg^'
+icon_svg: <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="60 0 163 163"><path d="M133.1 71.4c2.3 0 5.2.9 7.4 3.2a9.2 9.2 0 0 1-3.3 15.1 11.8 11.8 0 0 1-12.1-1.4 9.2 9.2 0 0 1 .5-15c2.1-1.4 4.4-1.9 7.5-1.9zm-.7 3.7h-.7l-.8.1c-2.3.5-4 1.7-4.6 4-.6 2.4-.1 4.6 1.9 6.2 1.5 1.2 3.4 1.5 5.3 1.2 4.2-.6 6.1-4.6 4.5-8.2-1-2.3-3.1-3.2-5.6-3.3zm-44.9 5a6 6 0 0 1 1.9 1.1c2.9 2.6 2 7.1-1.7 8.5-1 .4-2.1.6-3.2.6h-7.8c-.5 0-.7-.2-.7-.7V72.3c0-.5.1-.7.7-.7h7.2c1.9 0 3.6.7 4.8 2.3a4.2 4.2 0 0 1-.6 5.8l-.6.4zm-7 4.6v1.8c0 .2 0 .4.3.4l3.3-.1c.9-.1 1.8-.3 2.2-1.2.6-1.2 0-2.5-1.5-2.9-1.3-.3-2.7-.1-4.1-.2-.3 0-.2.2-.2.4v1.8zm0-7.6v1.6c0 .2 0 .3.2.3.9 0 1.9.1 2.8-.1 1-.3 1.6-1.1 1.6-2.1-.1-.9-.8-1.6-1.9-1.7h-2.4c-.3 0-.3.1-.3.3v1.7zm31.3 6.2v-2.6c.1-.3.3-.3.5-.3h6.4c.4 0 .6.2.6.6v7c0 .4-.2.6-.5.8a14.4 14.4 0 0 1-14.1-.1 9.3 9.3 0 0 1 .8-15.9c3.5-1.8 7.1-1.8 10.7-.8l2.1.7c.4.1.6.4.6.8V76c0 .7-.2.8-.9.5a15 15 0 0 0-6.9-1.5 6 6 0 0 0-4.9 2.5c-2.5 3.4-.7 8 3.4 8.9 1.6.4 3.2.3 4.8-.2.3-.1.4-.2.4-.6v-1.5c0-.3-.1-.4-.4-.4h-2.1c-.2 0-.5.1-.6-.3.2.1.2 0 .1-.1zM195 81v-8.6c0-.8 0-.8.8-.8h11.3c.8 0 .8 0 .8.8v2c0 .6-.2.7-.7.7H200c-.4 0-.6.1-.6.5v2.9c0 .4.1.5.5.5h7c.5 0 .7.2.7.7v2c0 .8-.1.8-.8.8H200c-.5 0-.6.1-.6.6v3.2c0 .4.1.5.5.5h7.3c.9 0 .9 0 .9.9v2c0 .4-.2.6-.6.6h-12c-.6 0-.6-.3-.6-.7l.1-8.6zm-10.8 1.5 2.5-5.7 2.1-4.8c.2-.4.4-.5.8-.5h3.2c.8 0 .9.2.5.9L187 86.2l-1.7 3.8c-.2.4-.4.5-.8.5-.9.1-1.3-.2-1.7-1l-8-17c-.4-.8-.3-1 .6-1h2.9c.5 0 .8.2 1 .6l4.6 9.9.3.5zm-26.4-10v13.3c-.3.2-.3.5 0 .7.1.4.3.2.5.2h7.9c.3 0 .6-.1.7.3v.3c-.1.1-.1.2 0 .2v2.2c0 .3-.2.4-.4.4h-12.8l-.1-.1V72c.1-.4.2-.6.6-.6h2.2c.3 0 .7.1 1-.1.5 0 .5.4.4.7-.2.4-.2.5 0 .5zm15.4 8.5v8.6c0 .7-.1.8-.8.8h-2.9c-.6 0-.7-.2-.7-.8V73c0-1.6-.1-1.4 1.4-1.5h2.3c.5 0 .7.2.7.7V81zm-75.3 0v8.5c0 .7 0 .8-.8.8h-2.9c-.7 0-.8 0-.8-.8V72.2c0-.5.2-.7.7-.7h3c.6 0 .7.2.7.7l.1 8.8z"/></svg>
```

Comment:
Better try to find something like that
https://www.svgrepo.com/svg/515017/bigo-live